content/cybersecurity/concepts/malware/terms/keylogger/keylogger.md

```diff
@@ -0,0 +1,57 @@
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+  - 'Malware'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+**Keyloggers** are a type of computer malware that records keystroke events on the keyboard and saves them to a log file, allowing the theft of sensitive data.
+Keystroke logging is one of the methods to obtain sensitive information like usernames, PINs, passwords and online activities from endpoint devices. An attacker can collect valuable data without entering into a strong database or file server using this method.
+
+## How Keyloggers Work
+
+Keyloggers, also known as keystroke loggers, record the keys hit on a device and save them to a file, which is then accessed by the person who deployed the malware.
+The main purpose of keyloggers is to tamper with the chain of events that occur when a key is pressed and information is displayed on the screen as a result of the keystroke.
+Keyloggers can be used for both lawful and illegitimate objectives,depending on the user who is utilizing it.
+
+Lawful applications include:
+
+- Developers and analysts studying user interaction with systems.
+- Employee monitoring.
+- Law enforcement or private investigators looking for evidence of an ongoing crime  or inappropriate behavior.
+- Quality assurance testers analyzing sources of system errors.
+
+On the other hand, unlawful use consists of cybercriminals using keylogging technology to capture identities, confidential intellectual property, passwords and any other marketable information.
+
+## Types of Keyloggers
+
+Keyloggers fall into four categories: software, hardware, wireless intercept, and acoustic. Although they differ in how they are implemented and how information is captured, these keyloggers have one thing in common. They store capture information in a log file.
+When software or hardware keyloggers are used, the log files are stored on the compromised machine.
+Remote capture technologies (i.e., wireless intercept and acoustic) typically store keystroke data on the collection device.
+
+### Software Keyloggers
+
+Software keyloggers capture keystroke information as it passes between the computer keyboard interface and the OS. They are implemented as traditional applications or are kernel-based. In almost all malicious instances of this type of keylogger, users participated in some way in the software’s installation.
+
+### Hardware Keyloggers
+
+A hardware keylogger is essentially a circuit located somewhere between the keyboard and the computer. The keylogger is connected directly to the PC and the keyboard to the keylogger.
+Another method is to install a keylogger circuit into a standard keyboard. This has the advantage of no visible evidence of user monitoring.
```

Comment:
I'm finding this a little unclear. Adding a circuit to a keyboard is actually creating physical evidence of monitoring. It is just very difficult to detect. Someone needs the expertise and the motivation to look for it. Is this sentence conveying something else?